src/pages/index.tsx

```diff
@@ -21,7 +22,14 @@ const HomePage: NextPage = () => {
       </Heading>
       <Heading as="h2" variant="h2">
         Fueled by{' '}
-        <Text as="span" variant="hero">
+        <Text
+          as="span"
+          css={css`
+            position: relative;
+            left: 50%;
+          `}
+          variant="hero"
+        >
```

Comment:
❓ Was this added for testing purposes?

```suggestion
        <Text as="span" variant="hero">
```

Note: the `css` import would need to be removed too.

Reply:
Yes, I modified to verify content area was expanding/collapsing correctly. Thought it should be fine since all of that content there is only placeholder for actual page.